Amz.py

```diff
@@ -89,7 +128,16 @@ def main():
                 driver.find_element(By.XPATH, '//span[@class="a-button-inner"]').click()
                 print('Please log in...')
 
+            # Checking to see if the giveaway has ended
+            try:
```

Comment:
Better to use our element_exists(..) function, then just check "if element" afterwards. Probably some examples of that in this code somewhere

Reply:
fixed